flink-yarn/src/test/java/org/apache/flink/yarn/YarnResourceManagerTest.java

```diff
@@ -526,6 +527,45 @@ public void testOnStartContainerError() throws Exception {
 		}};
 	}
 
+	@Test
+	public void testGetCpuCoresCommonOption() throws Exception {
+		final Configuration configuration = new Configuration();
+		configuration.setDouble(TaskManagerOptions.CPU_CORES, 1.0);
```

Comment:
It would be nice also to test rounding up to 1.0 and next greater integer for double values of `TaskManagerOptions.CPU_CORES`. Also, overrunning of `Integer.MAX_VALUE`